libraries/lib-components/ConfigInterface.h

```diff
@@ -53,10 +54,10 @@ differentiates between private and shared config.  It should probably be replace
 with a Shuttle.
 
 *******************************************************************************************/
-class AUDACITY_DLL_API ConfigClientInterface /* not final */
+class COMPONENTS_API ConfigClientInterface /* not final */
```

Comment:
Also I find very interesting your idea about use ShuttleParams in Config interfaces.

Reply:
Again, not my idea :-)

I'm not always clear on the purpose of all these abstract interfaces I didn't write, but I move some files around to manage dependencies.

This interface doesn't mention Shuttles at all, in fact.  It's making a sort of abstract interface around something like a configuration file.